### DISCUSSION TITLE
Possibility to add other "status tags" to the project?

### DISCUSSION THREAD

**Discussion by cives** [vkbo/novelWriter#1089] [Ideas]
Hi, Veronica and everyone,
I have been trying several writing tools and none worked with my chaotic "methodological pantser" style, as put by Ellen Brock on its YT channel.
Until now, when I found this wonderful piece of coding art.
I am structuring my WiP novel, which, due to my style, I am planning and revising constantly as I write it.
Therefore, having a overview of what I am writing and some tags has been proven great to know what I am writing and what I should change or adapt while I develop the story. Or better said, the characters write the story themselves, with me trying to add some order a posteriori to their own self-writing. :-D
Briefly, besides having the possibility to mark the folders, chapters, scenes and notes as drafts or finished, it would be perfect for me to have the option to add AT THE SAME TIME other "status tags" such as "1st person", "3rd person", or "action", "emotions", "plot turns" or other elements of the structure. That way, I could see how the story is at a glance in terms of such elements that need to be adjusted. 
I include a screenshot of my actual status, where I added some additional "status tags", but now I have to choose between marking a scene as "1st draft" OR "1st person" POV. What I would love is to be able to mark the scen as "1st draft" AND "1st person. And other additional statuses, eventually.
If that would be possible, it would be nice.
In the meantime, I can try to mark the containing chapter as "draft" and the scenes as pov or plot turns, etc.
I must add that I already created folders and tags for that purpose, as you can see as well in the screenshot, but they do not show in the overview tree panel, to check the consistence and coherence between styles and scenes and chapters.
Thank you again for your wonderful work here.
Antonio

![Screenshot_5](https://user-images.githubusercontent.com/8943127/174452022-9b4fa8cb-d842-4fdc-af97-1a7556642bf6.png)

**Comment by vkbo** [comment]
Hi!

First of all, it's always fun to see what different ways people use my project. Thank you for sharing your process. I am a structured writer myself, and the features of novelWriter certainly reflects that. Without, I hope, being too restrictive in forcing you to write that way.

The status flag is a tool to use in whatever way you wish. The limitation here is of course that you can only give your documents one status at a time. I can see a couple of ways to implement an option to choose multiple status flags, which may solve your use case, but I'm a bit concerned that it would quickly look cluttered.

However, there is another issue the status flag doesn't really solve, and that is you cannot set the status of individual story elements. It assumes you have a document for each component, like chapter and scene.

Perhaps there should be a way to define in the text the various flags for each story section (under each header), and let them show up in the Outline panel, which you can already customise to show whatever you want. Technically, you can already do this using the `@custom` tag. But perhaps it would be useful to add a `@flag` keyword that can be used for anything you want. The target of them could be added as another tab in project settings like the one for status and importance.

Does that sound like a workable solution?

Incidentally, in the next release there will be a lot of changes to the project tree and the novel tree. In addition to a visual update, there are a lot of new features and options on how to display content. Here are some screenshot from the work in progress:
![image](https://user-images.githubusercontent.com/1619840/174498355-2edc71ce-a8ed-47f1-a473-998e2ec06769.png)
![image](https://user-images.githubusercontent.com/1619840/174498371-1aa7000a-2f91-4a7f-b43b-73d9d39fa854.png)

**Reply by cives** [reply]
Thank you, Veronica:
What you suggest I think it's perfect. I don't know if showing one more column in the tree/outline view for the status flag would make it very cluttered, but you could choose to use it or keep it as now, maybe.
I already use the tags, including the few custom ones, to individualize each different element. That way I didn't use up-to-date a file "cover" for the chapter name and the metadata, but I created a file for every likely scene-to-be and marked it with is own metadata.
The issue is that those metadata cannot be seen from an overview perspective.
Only later on, when I have worked more on the scenes, I will be able to know where everything will go, in which chapter, if merging scenes, deleting them, or maybe I will need to change a PoV narrator in a scene that doesn't go along with the rest of that chapter. For instance. 
With a quick glance to one or more columns of the tree view I could check very fast that status and if I am being coherent.
That's why having a view of some key metadata from the broad overview perspective would be essential to me. If possible. If not, I am delighted with your exceptional work anyway.
Regards,
Antonio

**Reply by vkbo** [reply]
The Outline panel is really the tool intended for checking consistency of the story. I plan to add more features to it later on. The tree views next to the editor will only be a minimal view. I will think a bit more about the idea of adding another optional flag.

**Reply by cives** [reply]
Sorry if I confound the tree with the outline view. This is the one I mean. If it could show more than one status indicator, it would be nice. That way I could see a broad picture of the scenes showing completion and PoV, or PoV plus another status such as plot turn/action/dialogue/a custom flag to show some special status...
If it's possible, if not, I will do as it is. Obviously.
Thanks.
Antonio
